### PR TITLE
Add experiment level tutorial step [fixes SCI-338] 

### PR DESCRIPTION
**Pull request by ZmagoD**
https://biosistemika.atlassian.net/browse/SCI-338

### CODE REVIEW

**Comment by jbargu** [comment]
You should disable clone/move/edit buttons on experiments, since now you can open them in the background.

**Comment by jbargu** [comment]
Dunno if this was reported but when you're on canvas tutorial (about to be saved), and then you drag a task and click save, yout get 403 (and you cannot edit modal. I think it was something similar reported.

File: config/locales/en.yml [hunk 1]
```diff
@@ -1457,27 +1457,29 @@ en:
...
+    create_experiment_html: "Inside of a project you can create experiments. To create one click on this button and edit experiment’s name and description."
+    edit_experiment_html: "By clicking on the down arrow <span class='caret'></span> of an experiment card you can choose between different actions. You can Edit details to the experiment, such as experiment name and description, Clone it inside of a project, where cloned experiment is identical to the original one, Move it from one project to another or Archive it in the archive <span class='glyphicon glyphicon-briefcase'></span>."
     canvas_overview_html: "A task is a basic unit of your experiment. Within a task, you can add protocols, results and add samples. You can connect tasks into workflows to assure traceability of your work. You can grab the task or entire workflow and move it in all directions on the canvas."
```
Comment (jbargu) [inline config/locales/en.yml]: After talking with marketing, could you remove the last sentence: "You can grab the task or entire workflow and move it in all directions on the canvas."

**Comment by jbargu** [comment]
Everthing looks muy buen. :ok_hand: :+1: :sheep:

File: config/locales/en.yml (hunk 1 above)
Comment (ZmagoD) [inline config/locales/en.yml]: Aj Aj captain

**Comment by jbargu** [comment]
This was fixed now, but on the project screen ('In the bottom row ...') you can now click on dropdown and edit.

**Comment by jbargu** [comment]
You fix that and it's good to go. :+1:

**Comment by ZmagoD** [comment]
You can open the drop down actions but you can't trigger it it was agreed with Darja to work in this way

**Comment by jbargu** [comment]
Yeah boii, works like a charm. :sheep: :it:

**Pull request merged**